nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_dataset.py

```diff
@@ -169,7 +169,7 @@ def _process_example(self, example):
         tokenized_text = pre_pad + self.tokenizer.text_to_ids(text)
         context_ids = pre_pad + self.tokenizer.text_to_ids(context)
         answer_ids = tokenized_text[len(context_ids) :]
-        total_ids = len(context_ids) + len(answer_ids)
+        total_ids = len(context_ids) + max(len(answer_ids), self.tokens_to_generate)
```

Comment:
Why is tokens to generate added here?  Does this not suffice? https://github.com/NVIDIA/NeMo/blob/20717bab25f5d0ebe4cbe7fbef614206aaa630e5/nemo/collections/nlp/data/language_modeling/megatron/gpt_sft_dataset.py#L265

Reply:
resolved offline